filelog.go

```diff
@@ -172,10 +184,17 @@ func NewFileLogWriter(fname string, rotate bool) *FileLogWriter {
 		format:   "[%D %T] [%L] (%S) %M",
 		rotate:   rotate,
 		rotateDateSuffix: false,
+		rotateOnStartup: true,
+		performStartupRotation: true,
 		errorWriter: os.Stderr,
 		started: false,
 	}
 
+	// If the file doesn't exist, there is nothing to rotate
+	if _, err := os.Lstat(w.filename); os.IsNotExist(err) {
```

Comment:
Yes, because we need to know if the file existed prior to the call to `openLogFile()`, because `openLogFile` will create it. If we don't have the `performStartupRotation` flag, we will rotate twice. I could change it to something like `currentFileExistedAtStartup`?